### PR TITLE
Update to CVXOPT version 1.1.9

### PR DESCRIPTION
**Pull request by martinandersen**
- New test script that runs tests in cvxopt/tests/
- Removed obsolete CVXOPT_SUITESPARSE_EXT_LIB from build.sh
- Patch no longer necessary

### CODE REVIEW

**Comment by martinandersen** [comment]
Update appears to be OK. @jakirkham, before I go ahead and merge, can you take a quick look?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,20 +10,11 @@ package:
...
-    # ( https://github.com/cvxopt/cvxopt/commit/007c1a0ea9df9a54e9f99cacbde56a6047518375 ).
-    # Also, see this PR to show how these were backported.
-    # ( https://github.com/cvxopt/cvxopt/pull/61 )
-    - cvxopt_PR_61.diff
+  md5: a56e7b23d76c2b5aaf3bea2a7c245ea7
 
 build:
   skip: true  # [win]
-  number: 204
+  number: 0
```
Comment (jjhelmus) [inline recipe/meta.yaml]: Should this be 200 as the package is a openblas variant?  @jakirkham would be the expert in this area.

Comment (jakirkham) [inline recipe/meta.yaml]: Correct this should be `200`.

**Comment by jakirkham** [comment]
Would be good to add `license_file: LICENSE` to the `about` section so as to package the license file while we are at it.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,20 +10,11 @@ package:
...
-  patches:
-    # Add a patch that fixes the linking order and adds a missing library.
-    # Also, fixes the environment variable used for SuiteSparse includes.
-    # See this commit
-    # ( https://github.com/cvxopt/cvxopt/commit/4e89755cb6aaf6493a49a3ad99dd5d5e007e1a5a ).
-    # ( https://github.com/cvxopt/cvxopt/commit/007c1a0ea9df9a54e9f99cacbde56a6047518375 ).
-    # Also, see this PR to show how these were backported.
-    # ( https://github.com/cvxopt/cvxopt/pull/61 )
-    - cvxopt_PR_61.diff
+  md5: a56e7b23d76c2b5aaf3bea2a7c245ea7
```
Comment (jakirkham) [inline recipe/meta.yaml]: We should be able to use `sha256` from PyPI now, which would be better.

**Comment by jakirkham** [comment]
Do we still require `suitesparse` 4.4.x or is it possible to move to 4.5.x?

**Comment by jakirkham** [comment]
Thanks @martinandersen. Looks pretty good. Couple of small questions and comments above. Sorry I hadn't looked at it until now.

**Comment by martinandersen** [comment]
I believe that it should be fine to use SuiteSparse 4.5.x.

**Comment by martinandersen** [comment]
The checks fail now—I'm not sure how to fix this.

**Comment by martinandersen** [comment]
SuiteSparse 4.5.x works locally, but fails on circleci.

**Comment by martinandersen** [comment]
Perhaps suitesparse should also appear under requirements/run in meta.yml? It should be possible to use 4.5.x.

**Comment by jakirkham** [comment]
Yeah I think `suitesparse` 4.5.x only builds shared libraries ATM. So would need to be in `run` as well as `build`. Right @grlee77?

**Comment by grlee77** [comment]
SuiteSparse 4.5.x still builds both static and dynamic librararies (see for example, the file existance tests in https://github.com/conda-forge/suitesparse-feedstock/blob/master/recipe/meta.yaml).  

In the recently merged conda-forge/suitesparse-feedstock#18, the recipe was changed to rely upon the `METIS` conda-forge package instead of building the version shipped with `SuiteSparse`.  Only a shared library version of `METIS` is in the conda-forge package.  

Not sure exactly what is going on here without spending more time, but it seems like CircleCI has passed now with SuiteSparse in the run dependencies.  Perhaps the dynamic libs are the ones that get used during the CVXOPT build when both are present?

**Comment by martinandersen** [comment]
It works with SuiteSparse 4.5.x as a run dependency, so I suppose we can go ahead and merge?

**Comment by jakirkham** [comment]
Thanks @martinandersen.

**Pull request merged**